app/controllers/imports_controller.rb

```diff
@@ -18,11 +18,11 @@ def create
   def import_from_csv(import_type, file, org)
     case import_type
     when "volunteer"
-      FileImporter.new(file, org).import_volunteers
+      UserImporter.new(file, org).import_volunteers
     when "supervisor"
-      FileImporter.new(file, org).import_supervisors
+      UserImporter.new(file, org).import_supervisors
     when "casa_case"
-      FileImporter.new(file, org).import_cases
+      CaseImporter.new(file, org).import_cases
     else
       {type: :error, message: "Something went wrong with the import, did you attach a csv file?"}
```

Comment:
The error is that a bad `import_type` has been passed, right? How about something like this for an error message?:

```
valid_import_types_string = %w{volunteer supervisor casa_case}.inspect
message = "Bad import type '#{import_type}'. Must be one of #{valid_import_types_string}."
```

Reply:
Or did we want to test that the provided file is there?:

```
unless File.file?(file)
  return { type: :error, message: "CSV import file not found: #{file}" }
```